src/cuga/cli/main.py

```
@@ -1368,6 +1371,149 @@ def start(
+            logger.info("🧹 Resetting config db for Travel Agent...")
+
+            reset_config_db()
+
+            # Build LLM config from environment (same as setup_demo_manage_config does)
+            llm_api_key_ref = ""
+            try:
+                from cuga.backend.secrets.seed import resolve_llm_api_key_ref
+
+                llm_api_key_ref = resolve_llm_api_key_ref()
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)